packages/frontend/src/api/useDialogById.tsx

```diff
@@ -51,14 +51,14 @@ export const getPropertyByCultureCode = (
     | Array<{
         __typename?: 'Localization';
         value: string;
-        cultureCode: string;
+        languageCode: string;
       }>
     | null
     | undefined,
 ): string => {
-  const defaultCultureCodes = ['nb-no', 'nb']; // TODO: Will be changed to -1 iso in the future
+  const defaultCultureCodes = ['nb']; // TODO: Will be changed to -1 iso in the future
```

Comment:
Kan fjerne comment her, er på set 1 nå 🚀 

```suggestion
  const defaultCultureCodes = ['nb'];
```

Reply:
Allright, var litt usikker på det seru! ;) Men fjerner 👍